executor_manager/services/sandbox/manager.py

```
@@ -0,0 +1,961 @@
+                        await self._handle_executor_dead(
+                            task_id_str, last_heartbeat or sandbox.last_activity_at
+                        )
+
+            except Exception as e:
+                logger.debug(
+                    f"[SandboxManager] Heartbeat check error for {task_id_str}: {e}"
+                )
+                continue
+
+    async def _handle_executor_dead(
+        self, sandbox_id: str, last_heartbeat: float
```

Ruff (0.14.10)

Unused method argument: `last_heartbeat`

(ARG002)